AirportDepartures.playground/Contents.swift

```diff
@@ -58,9 +128,18 @@ import UIKit
 //:     Destination: Los Angeles Airline: Delta Air Lines Flight: KL 6966 Departure Time:  Terminal: 4 Status: Canceled
 //:     Destination: Rochester Airline: Jet Blue Airways Flight: B6 586 Departure Time: 1:26 PM Terminal:  Status: Scheduled
 //:     Destination: Boston Airline: KLM Flight: KL 6966 Departure Time: 1:26 PM Terminal: 4 Status: Scheduled
-
-
-
+func printDepartures2(departureBoard: DepartureBoard) {
+    for flight in departureBoard.departureFlights {
+        let flightTerminal = flight.terminal ?? ""
+        if let departureTime = flight.time {
+        print("Destination: \(flight.destination) Airline: \(flight.airline) Flight: \(flight.flightID) Departure Time:  \(departureTime) Terminal: \(flightTerminal) Status: \(flight.status.rawValue)")
+        } else {
+           print("Destination: \(flight.destination) Airline: \(flight.airline) Flight: \(flight.flightID) Departure Time:   Terminal: \(flightTerminal) Status: \(flight.status.rawValue)")
+        }
+    }
+}
```

Comment:
Love what you came up with here. Very clean with only two print statements and using the nil coalescing operator.